doc/subsystems/networking/net-app-api.rst

```diff
@@ -0,0 +1,199 @@
+.. _net_app_api:
+
+Network Application API
+#######################
+
+The Network Application API allows applications to
```

Comment:
Also maybe "Network Application (net-app) API", since "net-app" is used as a short form of the name below?